to-cfn-params.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# script to convert Codepipeline format parameters to cloudformation stack parameter files
+# usage:
+#   $ cat codepipeline-params.json | to-cfn-params.sh
```

Comment:
Question, from a usage PoV - why not `to-cfn-params.sh codepipeline-params.json`?

Reply:
because it makes assumptions about what you want the output to be called
- you don't want to overwrite the original
- should filename.json become filename-cli.json ?
- what happens when the filename has no extension?

I just wanted to keep the script as simple as possible

Reply:
Following discussion - as a filter script, it makes it more flexible.  e.g. input may not be from files.

👍